lib/rules/lint-attribute-indentation.js

```diff
@@ -408,6 +424,8 @@ module.exports = class AttributeSpacing extends Rule {
       let result = {
         maxLength: 80,
         indentation: 2,
+        blocklessMustacheEnd: 'new-line',
+        elementsEnd: 'new-line',
```

Comment:
Since this is specifically talking about how to end the _opening_ of the element, how about `endElementOpen`?

Reply:
See above, I like the naming, but I would keep the `*EndOpen` for consistency, what do you think?

Reply:
ya, seems good (the main point I was making was to make it clear that we are talking about the "end" of the "open" as opposed to the actual closing tag)